eng/templates/stages/mirror.yml

```
@@ -13,16 +13,18 @@ parameters:
 - name: isDryRun
   type: boolean
   default: false
+- name: dotnetStagingPipelineResource
+  type: string
 
 stages:
 - stage: Mirror
-  displayName: Mirror branch to DSP
+  displayName: Mirror to DSP
   dependsOn:
   - PreRelease
   - MirrorApproval
 
   jobs:
-  - job: Mirror
+  - job: MirrorSource
```

After consolidating the jobs, I think the original name was better.
```suggestion
  - job: Mirror
```

---

@mmitche - no sure if you saw this comment before merging.

---

ahh missed that. One sec will open a follow up.